test/api/v3/integration/groups/POST-groups_invite.test.js

```diff
@@ -0,0 +1,197 @@
+import {
+  generateUser,
+  translate as t,
+} from '../../../../helpers/api-integration.helper';
```

Comment:
This will need to change to ../../../../helpers/api-v3-integration.helper';

Also, we need a line break.